### ISSUE TITLE
Building fails with criterion dependency

### ISSUE THREAD

**Issue by jabley** [bos/pronk#1]
Sorry, very basic haskell newbie question:

```
$ cabal install
Resolving dependencies...
cabal: cannot configure pronk-0.1.0.0. It requires criterion >=0.5.1.0
There is no available version of criterion that satisfies >=0.5.1.0
```

http://hackage.haskell.org/package/criterion only has 0.5.0.10. What am I missing?

**Comment by arnihermann** [comment]
I have the same issue, I searched but couldn't find criterion 0.5.1.0 anywhere. I also tried just for kicks to change criterion dependency version to 0.5.0.10. The result was a compilation error:

```
[1 of 4] Compiling Network.HTTP.LoadTest.Types ( lib/Network/HTTP/LoadTest/Types.hs, dist/build/Network/HTTP/LoadTest/Types.o )

lib/Network/HTTP/LoadTest/Types.hs:40:15:
    Not in scope: data constructor `Request'

lib/Network/HTTP/LoadTest/Types.hs:50:21:
    Not in scope: data constructor `Request'
```

**Comment by tom-pang** [comment]
+1 ing this.

**Comment by vlm** [comment]
[vlm@nala:~/work/github/pronk (master)]> cabal update
Downloading the latest package list from hackage.haskell.org
[vlm@nala:~/work/github/pronk (master)]> cabal install criterion
Resolving dependencies...
No packages to be installed. All the requested packages are already installed.
If you want to reinstall anyway then use the --reinstall flag.
[vlm@nala:~/work/github/pronk (master)]> cabal build
Preprocessing library pronk-0.1.0.0...
Preprocessing executables for pronk-0.1.0.0...
Building pronk-0.1.0.0...
[2 of 4] Compiling Network.HTTP.LoadTest.Analysis ( lib/Network/HTTP/LoadTest/Analysis.hs, dist/build/Network/HTTP/LoadTest/Analysis.o )

lib/Network/HTTP/LoadTest/Analysis.hs:12:28:
    Module `Criterion.Analysis' does not export`SampleAnalysis'

lib/Network/HTTP/LoadTest/Analysis.hs:12:44:
    Module `Criterion.Analysis' does not export`analyseSample'

lib/Network/HTTP/LoadTest/Analysis.hs:12:59:
    Module `Criterion.Analysis' does not export`scale'
[vlm@nala:~/work/github/pronk (master)]>

**Comment by vlm** [comment]
The solution is to install this first: https://github.com/bos/criterion

**Comment by jabley** [comment]
That fails to install for me:

``` shell
$ git clone https://github.com/bos/criterion.git
$ pushd criterion
$ cabal install criterion
Resolving dependencies...
Downloading attoparsec-0.9.1.2...
Configuring attoparsec-0.9.1.2...
Preprocessing library attoparsec-0.9.1.2...
Building attoparsec-0.9.1.2...

Data/Attoparsec/Number.hs:18:8:
    Could not find module `Control.DeepSeq':
      There are files missing in the `deepseq-1.1.0.2' package,
      try running 'ghc-pkg check'.
      Use -v to see a list of the files searched for.
Downloading erf-2.0.0.0...
Configuring erf-2.0.0.0...
Preprocessing library erf-2.0.0.0...
Building erf-2.0.0.0...
[1 of 1] Compiling Data.Number.Erf  ( src/Data/Number/Erf.hs, dist/build/Data/Number/Erf.o )
Registering erf-2.0.0.0...
Installing library in /Users/jabley/.cabal/lib/erf-2.0.0.0/ghc-7.0.3
Registering erf-2.0.0.0...
Downloading monad-par-0.1.0.1...
Configuring monad-par-0.1.0.1...
Preprocessing library monad-par-0.1.0.1...
Building monad-par-0.1.0.1...

Control/Monad/Par.hs:99:8:
    Could not find module `Control.DeepSeq':
      There are files missing in the `deepseq-1.1.0.2' package,
      try running 'ghc-pkg check'.
      Use -v to see a list of the files searched for.
Downloading primitive-0.4.0.1...
Configuring primitive-0.4.0.1...
Preprocessing library primitive-0.4.0.1...
Building primitive-0.4.0.1...
[1 of 8] Compiling Data.Primitive.Internal.Compat ( Data/Primitive/Internal/Compat.hs, dist/build/Data/Primitive/Internal/Compat.o )
[2 of 8] Compiling Data.Primitive.MachDeps ( Data/Primitive/MachDeps.hs, dist/build/Data/Primitive/MachDeps.o )
[3 of 8] Compiling Control.Monad.Primitive ( Control/Monad/Primitive.hs, dist/build/Control/Monad/Primitive.o )
[4 of 8] Compiling Data.Primitive.Types ( Data/Primitive/Types.hs, dist/build/Data/Primitive/Types.o )
[5 of 8] Compiling Data.Primitive.Array ( Data/Primitive/Array.hs, dist/build/Data/Primitive/Array.o )
[6 of 8] Compiling Data.Primitive.ByteArray ( Data/Primitive/ByteArray.hs, dist/build/Data/Primitive/ByteArray.o )
[7 of 8] Compiling Data.Primitive.Addr ( Data/Primitive/Addr.hs, dist/build/Data/Primitive/Addr.o )
[8 of 8] Compiling Data.Primitive   ( Data/Primitive.hs, dist/build/Data/Primitive.o )
Registering primitive-0.4.0.1...
Installing library in /Users/jabley/.cabal/lib/primitive-0.4.0.1/ghc-7.0.3
Registering primitive-0.4.0.1...
Downloading text-0.11.1.5...
Configuring text-0.11.1.5...
Preprocessing library text-0.11.1.5...
Building text-0.11.1.5...

Data/Text.hs:191:8:
    Could not find module `Control.DeepSeq':
      There are files missing in the `deepseq-1.1.0.2' package,
      try running 'ghc-pkg check'.
      Use -v to see a list of the files searched for.
Downloading vector-0.9...
Configuring vector-0.9...
Preprocessing library vector-0.9...
Building vector-0.9...
[ 1 of 30] Compiling Data.Vector.Storable.Internal ( Data/Vector/Storable/Internal.hs, dist/build/Data/Vector/Storable/Internal.o )
[ 2 of 30] Compiling Data.Vector.Fusion.Util ( Data/Vector/Fusion/Util.hs, dist/build/Data/Vector/Fusion/Util.o )
[ 3 of 30] Compiling Data.Vector.Fusion.Stream.Size ( Data/Vector/Fusion/Stream/Size.hs, dist/build/Data/Vector/Fusion/Stream/Size.o )

Data/Vector/Fusion/Stream/Size.hs:30:10:
    Warning: No explicit method nor default method for `*'
    In the instance declaration for `Num Size'

Data/Vector/Fusion/Stream/Size.hs:30:10:
    Warning: No explicit method nor default method for `abs'
    In the instance declaration for `Num Size'

Data/Vector/Fusion/Stream/Size.hs:30:10:
    Warning: No explicit method nor default method for `signum'
    In the instance declaration for `Num Size'
[ 4 of 30] Compiling Data.Vector.Internal.Check ( Data/Vector/Internal/Check.hs, dist/build/Data/Vector/Internal/Check.o )
[ 5 of 30] Compiling Data.Vector.Fusion.Stream.Monadic ( Data/Vector/Fusion/Stream/Monadic.hs, dist/build/Data/Vector/Fusion/Stream/Monadic.o )
Loading package ghc-prim ... linking ... done.
Loading package integer-gmp ... linking ... done.
Loading package base ... linking ... done.
Loading package primitive-0.4.0.1 ... linking ... done.
Loading package ffi-1.0 ... linking ... done.
[ 6 of 30] Compiling Data.Vector.Fusion.Stream ( Data/Vector/Fusion/Stream.hs, dist/build/Data/Vector/Fusion/Stream.o )
[ 7 of 30] Compiling Data.Vector.Fusion.Stream.Safe ( Data/Vector/Fusion/Stream/Safe.hs, dist/build/Data/Vector/Fusion/Stream/Safe.o )
[ 8 of 30] Compiling Data.Vector.Fusion.Stream.Monadic.Safe ( Data/Vector/Fusion/Stream/Monadic/Safe.hs, dist/build/Data/Vector/Fusion/Stream/Monadic/Safe.o )
[ 9 of 30] Compiling Data.Vector.Generic.Mutable ( Data/Vector/Generic/Mutable.hs, dist/build/Data/Vector/Generic/Mutable.o )
[10 of 30] Compiling Data.Vector.Generic.Mutable.Safe ( Data/Vector/Generic/Mutable/Safe.hs, dist/build/Data/Vector/Generic/Mutable/Safe.o )
[11 of 30] Compiling Data.Vector.Generic.Base ( Data/Vector/Generic/Base.hs, dist/build/Data/Vector/Generic/Base.o )
[12 of 30] Compiling Data.Vector.Generic.New ( Data/Vector/Generic/New.hs, dist/build/Data/Vector/Generic/New.o )
[13 of 30] Compiling Data.Vector.Generic ( Data/Vector/Generic.hs, dist/build/Data/Vector/Generic.o )
[14 of 30] Compiling Data.Vector.Generic.Safe ( Data/Vector/Generic/Safe.hs, dist/build/Data/Vector/Generic/Safe.o )
[15 of 30] Compiling Data.Vector.Generic.New.Safe ( Data/Vector/Generic/New/Safe.hs, dist/build/Data/Vector/Generic/New/Safe.o )
[16 of 30] Compiling Data.Vector.Primitive.Mutable ( Data/Vector/Primitive/Mutable.hs, dist/build/Data/Vector/Primitive/Mutable.o )
[17 of 30] Compiling Data.Vector.Primitive ( Data/Vector/Primitive.hs, dist/build/Data/Vector/Primitive.o )
[18 of 30] Compiling Data.Vector.Primitive.Safe ( Data/Vector/Primitive/Safe.hs, dist/build/Data/Vector/Primitive/Safe.o )
[19 of 30] Compiling Data.Vector.Primitive.Mutable.Safe ( Data/Vector/Primitive/Mutable/Safe.hs, dist/build/Data/Vector/Primitive/Mutable/Safe.o )
[20 of 30] Compiling Data.Vector.Storable.Mutable ( Data/Vector/Storable/Mutable.hs, dist/build/Data/Vector/Storable/Mutable.o )
[21 of 30] Compiling Data.Vector.Storable ( Data/Vector/Storable.hs, dist/build/Data/Vector/Storable.o )
[22 of 30] Compiling Data.Vector.Unboxed.Base ( Data/Vector/Unboxed/Base.hs, dist/build/Data/Vector/Unboxed/Base.o )
[23 of 30] Compiling Data.Vector.Unboxed ( Data/Vector/Unboxed.hs, dist/build/Data/Vector/Unboxed.o )
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XD} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XD} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XD} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XH} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XD} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Xz} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XD} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XH} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XT} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XT} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XT} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XX} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XT} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XT} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26XX} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Y9} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s26Y9} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
[24 of 30] Compiling Data.Vector.Unboxed.Safe ( Data/Vector/Unboxed/Safe.hs, dist/build/Data/Vector/Unboxed/Safe.o )
[25 of 30] Compiling Data.Vector.Unboxed.Mutable ( Data/Vector/Unboxed/Mutable.hs, dist/build/Data/Vector/Unboxed/Mutable.o )
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA8} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA8} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA8} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAc} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA8} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA4} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uA8} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAc} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAo} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAo} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAo} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAs} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAo} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAo} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAs} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAE} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
SpecConstr
    Function `$w$j{v s2uAE} [lid]'
      has one call pattern, but the limit is 0
    Use -fspec-constr-count=n to set the bound
    Use -dppr-debug to see specialisations
[26 of 30] Compiling Data.Vector.Unboxed.Mutable.Safe ( Data/Vector/Unboxed/Mutable/Safe.hs, dist/build/Data/Vector/Unboxed/Mutable/Safe.o )
[27 of 30] Compiling Data.Vector.Mutable ( Data/Vector/Mutable.hs, dist/build/Data/Vector/Mutable.o )
[28 of 30] Compiling Data.Vector      ( Data/Vector.hs, dist/build/Data/Vector.o )
[29 of 30] Compiling Data.Vector.Safe ( Data/Vector/Safe.hs, dist/build/Data/Vector/Safe.o )
[30 of 30] Compiling Data.Vector.Mutable.Safe ( Data/Vector/Mutable/Safe.hs, dist/build/Data/Vector/Mutable/Safe.o )
Registering vector-0.9...
Installing library in /Users/jabley/.cabal/lib/vector-0.9/ghc-7.0.3
Registering vector-0.9...
Downloading mwc-random-0.10.0.1...
Configuring mwc-random-0.10.0.1...
Preprocessing library mwc-random-0.10.0.1...
Building mwc-random-0.10.0.1...
[1 of 1] Compiling System.Random.MWC ( System/Random/MWC.hs, dist/build/System/Random/MWC.o )
Registering mwc-random-0.10.0.1...
Installing library in /Users/jabley/.cabal/lib/mwc-random-0.10.0.1/ghc-7.0.3
Registering mwc-random-0.10.0.1...
Downloading vector-algorithms-0.5.3...
Configuring vector-algorithms-0.5.3...
Preprocessing library vector-algorithms-0.5.3...
Building vector-algorithms-0.5.3...
[1 of 9] Compiling Data.Vector.Algorithms.Common ( Data/Vector/Algorithms/Common.hs, dist/build/Data/Vector/Algorithms/Common.o )
[2 of 9] Compiling Data.Vector.Algorithms.Radix ( Data/Vector/Algorithms/Radix.hs, dist/build/Data/Vector/Algorithms/Radix.o )
[3 of 9] Compiling Data.Vector.Algorithms.Search ( Data/Vector/Algorithms/Search.hs, dist/build/Data/Vector/Algorithms/Search.o )
[4 of 9] Compiling Data.Vector.Algorithms.Optimal ( Data/Vector/Algorithms/Optimal.hs, dist/build/Data/Vector/Algorithms/Optimal.o )
[5 of 9] Compiling Data.Vector.Algorithms.Insertion ( Data/Vector/Algorithms/Insertion.hs, dist/build/Data/Vector/Algorithms/Insertion.o )
[6 of 9] Compiling Data.Vector.Algorithms.AmericanFlag ( Data/Vector/Algorithms/AmericanFlag.hs, dist/build/Data/Vector/Algorithms/AmericanFlag.o )
[7 of 9] Compiling Data.Vector.Algorithms.Heap ( Data/Vector/Algorithms/Heap.hs, dist/build/Data/Vector/Algorithms/Heap.o )
[8 of 9] Compiling Data.Vector.Algorithms.Intro ( Data/Vector/Algorithms/Intro.hs, dist/build/Data/Vector/Algorithms/Intro.o )
[9 of 9] Compiling Data.Vector.Algorithms.Merge ( Data/Vector/Algorithms/Merge.hs, dist/build/Data/Vector/Algorithms/Merge.o )
Registering vector-algorithms-0.5.3...
Installing library in
/Users/jabley/.cabal/lib/vector-algorithms-0.5.3/ghc-7.0.3
Registering vector-algorithms-0.5.3...
cabal: Error: some packages failed to install:
aeson-0.3.2.12 depends on text-0.11.1.5 which failed to install.
attoparsec-0.9.1.2 failed during the building phase. The exception was:
ExitFailure 1
blaze-builder-0.3.0.1 depends on text-0.11.1.5 which failed to install.
blaze-textual-0.2.0.4 depends on text-0.11.1.5 which failed to install.
criterion-0.5.0.10 depends on text-0.11.1.5 which failed to install.
double-conversion-0.2.0.1 depends on text-0.11.1.5 which failed to install.
hashable-1.1.2.1 depends on text-0.11.1.5 which failed to install.
monad-par-0.1.0.1 failed during the building phase. The exception was:
ExitFailure 1
statistics-0.9.0.0 depends on text-0.11.1.5 which failed to install.
text-0.11.1.5 failed during the building phase. The exception was:
ExitFailure 1
unordered-containers-0.1.4.3 depends on text-0.11.1.5 which failed to install.
[exited with 1]
```

I suspect there are problems with my install of haskell, which was installed via Homebrew. `ghc-pkg check` reveals a lot of problems. I'll take that up on the Homebrew project.

**Comment by vlm** [comment]
Yes, upgrade Haskell-Platform.

**Comment by bos** [comment]
OK, I believe the components of this are now fixed.
- There's a new version of `criterion` on Hackage (don't forget to run `cabal update`!)
- The version dependency on `http-enumerator` is now fixed
- I can't do much about the `deepseq` part, I think

If nobody has any updates, I'll close this issue in a few days. Thanks!

**Comment by jabley** [comment]
``` bash
pushd /path/to/pronk-src
cabal update
cabal install --reinstall criterion
cabal configure
cabal install
```

works for me

**Comment by vlm** [comment]
Works for me.

**Comment by bos** [comment]
I believe this is fixed now.
